apps/hub/src/app/_providers/wagmi-provider.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+
+import { createConfig, http, WagmiProvider as WagmiProviderBase } from 'wagmi'
+import { mainnet } from 'wagmi/chains'
+import { injected, metaMask, walletConnect } from 'wagmi/connectors'
+
+import { clientEnv } from '~/config/env.client.mjs'
```

Comment:
```suggestion
import { clientEnv } from '~constants/env.client.mjs'
```
Config does not exist under `apps/hub/src/`